src/components/Plot.tsx

```diff
@@ -211,72 +212,99 @@ export function Plot(props: PlotProps) {
     <plotContext.Provider value={plotContextValue}>
       <plotDispatchContext.Provider value={dispatch}>
         <LegendProvider>
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width={width}
-            height={height}
-            style={finalSvgStyle}
-            id={svgId}
-            className={svgClassName}
-          >
-            {/* Plot viewport */}
-            <TransparentRect
+          <div style={{ position: 'relative', width: width, height: height }}>
+            {/* Annottions & Tracking layer */}
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
               width={width}
               height={height}
-              style={plotViewportStyle}
-            />
-
-            {/* Series viewport */}
-            <g transform={`translate(${leftOffset}, ${topOffset})`}>
+              style={{
+                ...finalSvgStyle,
+                position: 'absolute',
+                top: '0',
+                left: '0',
+                zIndex: '10',
```

Comment:
I think if you swap the elements' zIndex becomes unnecessary?

Reply:
going to try

Reply:
fixed